products/dev_alternate_products.md

```diff
@@ -2,65 +2,96 @@
-Details particular to an IntelliJ Platform-based IDE is described on the individual product pages in Part VIII.
+Project configuration attributes common to projects targeting products other than IntelliJ IDEA are described here. 
+Details particular to an IntelliJ Platform-based product are described on the individual product pages in Part VIII.
+All of the Gradle configuration attributes described here are discussed in-depth on the [Configuring Gradle for IntelliJ Platform Plugins](/tutorials/build_system/gradle_guide.md) and the `gradle-intellij-plugin` [README](https://github.com/JetBrains/gradle-intellij-plugin/blob/master/README.md) pages.
 
 * bullet list
 {:toc}
 
-## Introduction 
-Plugins can extend or add functionality to IDEs based on the IntelliJ Platform. 
-Plugins can be written in Kotlin or Java, or a mix of both, and are created using IntelliJ IDEA.
-Once completed, plugins can be packaged and distributed at [plugins.jetbrains.com](https://plugins.jetbrains.com).
-
-## Configuring Plugin Projects Targeting Alternate IntelliJ Platform-Based IDEs
+## Getting Started
 To create a new Gradle plugin project, follow the tutorial on the [Getting Started with Gradle](/tutorials/build_system/prerequisites.md) page.
 The tutorial produces a skeleton Gradle project suitable to use as a starting point.
-Some minor modifications are needed, as described below, and on the individual product pages in Part VIII.
 
-Understanding the relationship between build numbers is critical when developing a plugin to target an IntelliJ Platform-based IDE:
-* _targetIDE_ is the (version-specific) IntelliJ Platform-based IDE in which the plugin is intended to run, such as PyCharm.
-  The path in Gradle plugin attribute [`runIde.ideaDirectory`](/tutorials/build_system/gradle_guide.md#running-against-alternate-versions-and-types-of-intellij-platform-based-ides) is set to the installed location of _targetIDE_.
+Modifications are needed to the skeleton project's `build.gradle` and `plugin.xml` files, as described below, and on the individual product pages in Part VIII.
+The `build.gradle` file is modified to specify the target product, which determines the APIs available during development.
+The `plugin.xml` file is modified to declare the plugin's dependency on modules or libraries.
+
+## Configuring Build.Gradle to Target Products other than IntelliJ IDEA 
+The best practice is to use the IntelliJ IDEA Gradle plugin `intellij.type` [attribute](https://github.com/JetBrains/gradle-intellij-plugin/blob/master/README.md#intellij-platform-properties) to specify the target product.
+For example, `PY` for PyCharm professional.
+Configuration using an `intellij.type` attribute is explained in the [Product-Specific Attribute](#configuring-plugin-projects-using-a-product-specific-attribute) section below.
+
+However, not all products have an `intellij.type` attribute defined by the IntelliJ IDEA Gradle plugin.
```

Comment:
the name of the plugin is "gradle-intellij-plugin" (multiple occurrences)

Reply:
I used "IntelliJ IDEA Gradle plugin" extensively in tutorials/buildsystem/prerequisites.md and gradle_config.md, but I'm agnostic.